services.md

```diff
@@ -0,0 +1,62 @@
+# services
+>
+> Auto generated on 2024-11-20 21:36:46 by
```

Comment:
Hmm I forgot what is our policy - do we store artifacts or not? 😉
I would suggest to not store it, but make it pop up when doing `make up-devel` or `make up-prod`. But I didn't think it through, and I am betting you did, so don't take that advice too seriously.

Reply:
This is not really an artifact. Note that the badges point to the HEAD files in master which are `upstream` but not in your local repo. I created the script to simplify the work.